Gruntfile.js

```diff
@@ -334,7 +334,8 @@ module.exports = function (grunt) {
           'app/scripts/services/simplesliderservice.js',
           'app/scripts/directives/simple-slider.js'
         ],
-        dest: 'dist/angular-simple-slider.js' },
+        dest: 'dist/angular-simple-slider.js'
+      },
```

Comment:
jshint complains about closing brace in the same line, do you still want me to revert that?